subgraphs/euler-finance/src/mappings/helpers.ts

```diff
@@ -599,37 +599,45 @@ export function rollbackRevenue(marketId: string, event: Liquidation, assetStatu
   const deltaTotalRevenueUSD = repayExtraUSD.times(RESERVE_FEE_SCALE).div(assetStatus.reserveFee.toBigDecimal());
   const deltaSupplySideRevenueUSD = deltaTotalRevenueUSD.minus(repayExtraUSD);
 
-  protocol.cumulativeTotalRevenueUSD = protocol.cumulativeTotalRevenueUSD.minus(deltaTotalRevenueUSD);
   protocol.cumulativeSupplySideRevenueUSD = protocol.cumulativeSupplySideRevenueUSD.minus(deltaSupplySideRevenueUSD);
+  protocol.cumulativeTotalRevenueUSD = protocol.cumulativeSupplySideRevenueUSD.plus(
```

Comment:
why are you setting total revenue to supply side revenue?

Reply:
total revenue = supply side + protocol side revenue

Reply:
true lol, i misread that